### PR TITLE
Build 0.1.10

### PR DESCRIPTION
**Pull request by mitcht**
Build latest changes into 0.1.10 just to keep the ball rolling.

FIXES:
Fixed issue in gruntfile.  Doing a math operation in jade on a possible non-existent file to try to get the size.  It will dynamically create size.  If you delete dist directure, grunt build would fail.  This is no longer the case.

### CODE REVIEW

File: starter/angular-xeditable/js/xeditable.js [hunk 1]
```diff
@@ -1040,6 +1052,11 @@ angular.module('xeditable').factory('editableFormController',
     }
   });
  
+  $rootScope.$on('closeEdit', function() {
```
Comment (mitcht) [inline starter/angular-xeditable/js/xeditable.js]: allows closing all 'edits' from outside of the directive.. .useful when you are using the xEditable system with other controls that 'open and close"

File: .gitignore [hunk 2]
```diff
@@ -3,3 +3,9 @@
 dev.html
 jsdoc.json
 /.idea
+node_modules
+dist
```
Comment (mitcht) [inline .gitignore]: don't include dist now. build will just make a fresh one if its missing.

File: Gruntfile.js [hunk 3]
```diff
@@ -128,14 +128,16 @@ module.exports = function(grunt) {
     return text;
   };
 
+
+
   var jadeData = {
     fs: require('fs'),
     md: marked,
     version: '<%= pkg.version %>',
-    size: Math.floor(fs.statSync('dist/js/xeditable.min.js').size / 1024),
```
Comment (mitcht) [inline Gruntfile.js]: Don't math on files that may or may not exist!

File: .gitignore (hunk 2 above)
Comment (eugef) [inline .gitignore]: Don't exclude `dist` from the git repo, as it is used by bower.

When you install with bower you should already have builded library without running any additional scripts

Comment (mitcht) [inline .gitignore]: Good catch. Our bower repo and source repo we use in house are separate.  I am just used to doing this.

**Comment by eugef** [comment]
Please squash all your commits in a few with meaningful commit messages and we are ready to merge.

**Comment by mitcht** [comment]
I can't get it out of conflict. It seems to be in conflict on the built .js file.

**Comment by mitcht** [comment]
I had a small issue where my branch merges up with master. I couldn't get the rebase to 'catch' that set of commits, so I had to leave 2.  If you are ok with that, good, otherwise suggestions?

**Comment by mitcht** [comment]
I've been trying to get them to squash properly for about 2 hours now. The timespan, coupled with my limited knowledge of github is preventing me from accomplishing this.

**Comment by mitcht** [comment]
OK THERE I GOT IT. Jeepers that was a pain.

**Comment by eugef** [comment]
Thanks for your commitment!

**Comment by ganqqwerty** [comment]
this is so cool, thanks! A must-have feature!!!

**Comment by mitcht** [comment]
Hah! Which one?  Theres a few in this release!

**Comment by ganqqwerty** [comment]
#6 , I've been waiting for it for so long...

**Comment by mitcht** [comment]
You could have always built it yourself :)  Fork, clone, npm install, bower install, grunt build.  Simple as that.

**Comment by richfergus** [comment]
Awesome - upgrading to (Version 0.1.12 Apr 14, 2016) fixed the "Form with editable elements should have `editable-form` attribute" requirement

**Pull request merged**